packages/wonder-blocks-progress-spinner/src/components/__docs__/circular-spinner.stories.js

```diff
@@ -0,0 +1,181 @@
+            <td>
+                <CircularSpinner size={"medium"} style={styles.distanced} />
+            </td>
+            <td>
+                <CircularSpinner size={"large"} style={styles.distanced} />
+            </td>
+        </tr>
+        <tr className={css(styles.darkBackground)}>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"xsmall"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"small"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"medium"}
+                    style={styles.distanced}
+                />
+            </td>
+            <td>
+                <CircularSpinner
+                    light={true}
+                    size={"large"}
+                    style={styles.distanced}
+                />
+            </td>
+        </tr>
+    </table>
+);
+
+Sizes.parameters = {
+    docs: {
+        storyDescription: `The available sizes for progress spinner are
+            \`"xsmall"\`, \`"small"\`, \`"medium"\`, and \`"large"\`.
+            This is set with the \`size\` prop.`,
+    },
+};
+
+export const Light: StoryComponentType = () => (
+    <View style={styles.darkBackground}>
+        <CircularSpinner light={true} />
+    </View>
+);
+
+Light.parameters = {
+    docs: {
+        storyDescription: `This is a progress spinner with its \`light\`
+            prop set to true. This is for use on dark backgrounds.`,
+    },
+};
+
+export const Inline: StoryComponentType = () => (
+    <Body>
+        Inline inside{" "}
+        <CircularSpinner size="xsmall" style={{display: "inline"}} /> some text.
+    </Body>
+);
+
+Inline.parameters = {
+    docs: {
+        storyDescription: `Circular spinners also work inline.`,
+    },
+};
+
+export const WithStyle: StoryComponentType = () => {
+    const spinnerStyle = {
+        border: `solid 5px ${Color.teal}`,
+        borderRadius: "50%",
+        backgroundColor: Color.offWhite,
+    };
+
+    return <CircularSpinner style={spinnerStyle} />;
+};
+
+WithStyle.parameters = {
+    docs: {
+        storyDescription: `\`<CircularSpinner>\` has \`style\` prop
+            that can be used to apply styles to the spinner's container.
+            Here, it has been given a style that includes \`border\` that
+            is \`solid 5px \${Color.teal}\` as well as a \`borderRadius\`
+            of \`50%\`.`,
+    },
+};
+
+const styles = StyleSheet.create({
+    darkBackground: {
+        background: Color.darkBlue,
+        padding: Spacing.xLarge_32,
+        width: "100%",
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    distanced: {
+        margin: Spacing.large_24,
+    },
```

Comment:
**style nit (non-blocking):** Instead of applying this style to each spinner, you could set `borderSpacing: 24` on the table itself.

Reply:
I'm not sure why, but this ends up looking very different 🤔 

<img width="497" alt="Screen Shot 2022-09-19 at 1 48 39 PM" src="https://user-images.githubusercontent.com/13231763/191113705-a63b72ee-e3c9-4d8b-94c0-7256802c9538.png">